### DISCUSSION TITLE
My hudl won't boot into recovery mode and I found the "flashing from any computer" instructions quite vague. Is it still possible to install the rom?

### DISCUSSION THREAD

**Discussion by FFFluoride** [remay/tmf-hudl#45] [Help]
My hudl doesn't go into recovery for some reason (I've tried dozens of times). I looked at the instructions for flashing from any computer and found them vague. Specifically, does the tablet itself have to be in recovery mode. Do I have to back up the files on the host computer? That's all but it sure doesn't feel good to be uncertain about these. Also this is my first time flashing an android rom like this, it's possible that I just missed something really obvious

**Comment by remay** [comment]
I'm afraid that it's impossible to know whether you can flash from a computer without understanding why it won't go into recovery mode - but it's probably your best bet.

Unfortunately it's difficult to be more specific with the instructions as getting *your* computer to boot from a USB stick will be specific to your computer.

If you can follow the instructions as best you can and let me know here which bit's you're having trouble with then I can try and give you some guidance.

Rob.

**Comment by remay** [comment]
> Specifically, does the tablet itself have to be in recovery mode.

No.  The tablet needs to be in a mode that allows flashing over USB which is covered in step 4 of the instructions (you did expand the steps by clicking on the little triangles?)
 
> Do I have to back up the files on the host computer?

No.   The hard drive on the host computer isn't touched (but you do have a backup anyway don't you).   The host boots from the USB stick and then runs entirely in memory.
